projects/whydense/gsc/sparse_speech_experiment.py

```diff
@@ -321,3 +325,24 @@ def load_datasets(self):
         self.test_loader = DataLoader(
             test_dataset, batch_size=self.batch_size, shuffle=False
         )
+
+    def _use_kwinner_local(self, model):
```

Comment:
I would like to move this logic into LeSparseNet() itself rather than in this class.